packages/react-next/src/components/Coachmark/Beak/Beak.tsx

```diff
@@ -8,78 +8,73 @@ import { RectangleEdge } from 'office-ui-fabric-react/lib/utilities/positioning'
 export const BEAK_HEIGHT = 10;
 export const BEAK_WIDTH = 18;
 
-export class Beak extends React.Component<IBeakProps, {}> {
-  constructor(props: IBeakProps) {
-    super(props);
+export const Beak = React.forwardRef((props: IBeakProps, forwardedRef: React.Ref<HTMLDivElement>) => {
+  const { left, top, bottom, right, color, direction = RectangleEdge.top } = props;
 
-    initializeComponentRef(this);
-  }
-
-  public render(): JSX.Element {
-    const { left, top, bottom, right, color, direction = RectangleEdge.top } = this.props;
+  React.useImperativeHandle(props.componentRef, () => ({}), []);
```

Comment:
Is this supposed to do anything? If not you should probably just remove the componentRef prop (and deprecate it in v7).